usaspending_api/recipient/management/sql/recipient_lookup/README.md

```diff
@@ -0,0 +1,7 @@
+# Overview
+
+These files populate and update `recipient_lookup` using the latest SAM data (in `duns`) and transaction data from `transaction_fabs` and `transaction_fpds`
+
+Useful single-line command to run all SQL files in correct order as a single file. Run from this directory.
+
+    ls -v *.sql | xargs cat | psql $DATABASE_URL -v ON_ERROR_STOP=1 -c '\timing' -f -
```

Comment:
If this is going to be the new ETL to update `recipient_lookup` should this be made similar to `load_subawards` where there is a management command that runs all of the necessary SQL and logs?

Reply:
I thought of that. I wasn't sure if it warranted a management command. If devs think it is worth it I can add one

Reply:
If it will be run more than once then I would personally put a vote in for it. It helps someone to understand what is going on if all they see is the command.

Reply:
So you're telling me that README files are worthless

Reply:
Only your README files, Tony... only yours... :-p

Reply:
Was thinking more so about if this is run in Jenkins.

Reply:
My intentions were not to use that bash in Jenkins, it was useful for devs to run all of the sql scripts locally.

Added new management command